packages/dashboard-core-plugins/src/panels/ChartPanel.tsx

```diff
@@ -458,6 +459,24 @@ export class ChartPanel extends Component<ChartPanelProps, ChartPanelState> {
       }))
   );
 
+  getWidgetPanelDescriptor = memoize(
+    (metadata: ChartPanelProps['metadata']): WidgetPanelDescriptor => {
+      let name = 'Chart';
+      if (isChartPanelTableMetadata(metadata)) {
+        name = metadata.table;
+      } else if (isChartPanelFigureMetadata(metadata)) {
+        name = metadata.figure ?? name;
+      } else {
+        name = metadata.name ?? name;
+      }
+      return {
+        ...metadata,
+        type: 'Chart',
```

Comment:
I think there's some confusion/murkiness here about what `type` is. The engine calls the widget a `Figure` (see `ChartPluginConfig`). The `WidgetPanel.widgetType` prop is actually used just to set the friendly name in the tooltip I believe, so this is not the right type for the descriptor

Reply:
True should clarify that. The `type` on the `WidgetDescriptor` should match the actual type, then I'll add a `displayType` optional prop instead for the friendly name.